package.json

```diff
@@ -5,6 +5,7 @@
   "bin": {
     "haul": "./bin/cli.js"
   },
+  "main": "src/index.js",
```

Comment:
Is this related to this PR somehow?

BTW configure `eslint.autoFixOnSave` in your editor instead of prettier so you don't format JSON files accidentally.

Reply:
It's we need to expose `createWebpackConfig` function.

Reply:
@satya164 As @jukben said it is required from node to know there to looks for `createWebpackConfig`.